packages/styling/src/styles/IconCodes.ts

```diff
@@ -2518,7 +2518,7 @@ export const IconCodes = {
   /**
    * Icon code with the value '\uE8AB'.
    */
-  switcher: '\uE8AB',
+  switch: '\uE8AB',
```

Comment:
This is a good question. At some point the name of the these icons changed, there were component examples in Fabric React that were looking for the delete icon that were broken when the name changed to trash.

We need a way to ensure that the icon names documented on the website match the icon names in Fabric React, whether that is a test or generating the icon codes list automatically.